.github/workflows/base.yml

```diff
@@ -77,8 +77,27 @@ jobs:
           name: reports_dir
           path: ./docs/reports
 
+  run_stubtests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}  # so that conda works
```

Comment:
After merging this, consider moving this `defaults` to the top, and removing duplicating `shell` parameters from elsewhere.

Reply:
Nice proposal, thanks!